pyproject.toml

```diff
@@ -56,6 +56,12 @@ build-backend = "flit_core.buildapi"
 [tool.flit.sdist]
 include = [".sqlfluff"]
 
+[tool.pytest.ini_options]
+minversion = "6.0"
+testpaths = [
+    "tests",
+]
+
```

Comment:
this came in while debugging the network issues, but i left it since limiting the search space of pytest can't hurt.